os-autoinst-openvswitch

```diff
@@ -53,7 +53,7 @@ sub init_switch ($self) {
     }
     system('ovs-vsctl', 'br-exists', $self->{BRIDGE});
 
-    for (my $timeout = $ENV{OS_AUTOINST_OPENVSWITCH_INIT_TIMEOUT} // ONE_MINUTE; $timeout > 0; --$timeout) {
+    for (my $timeout = $ENV{OS_AUTOINST_OPENVSWITCH_INIT_TIMEOUT} // 5 * ONE_MINUTE; $timeout > 0; --$timeout) {
```

Comment:
```suggestion
    for (my $timeout = $ENV{OS_AUTOINST_OPENVSWITCH_INIT_TIMEOUT} // (5 * ONE_MINUTE); $timeout > 0; --$timeout) {
```

Reply:
Done. I also extracted the whole thing into a constant now